lib/improver/tests/utilities/test_mathematical_operations.py

```diff
@@ -333,7 +384,7 @@ def test_start_point_positive(self):
             result.coord("height").points, np.array([20.]))
         self.assertArrayAlmostEqual(result.data, expected)
 
-    def test_start_point_negative(self):
+    def test_start_point_negative_direction(self):
```

Comment:
This doc-string and the one above appear to be in conflict (as do all the positive/negative test doc-strings below). Both say that integrating downwards in a height coordinate is equivalent to their direction (both positive and negative). I think they are both valid as the cubes have been ordered differently, but the statement reads as if we are making a general point about integration in a height coordinate. Could we just tweak them to something like:
```
(equivalent to integrating downwards for the height coordinate in the input cube)
```
which makes it sound more specific to this test case?

Reply:
I've amended this docstring and the docstrings for the other tests as suggested.